Framework/DataObjects/src/MDHistoWorkspace.cpp

```diff
@@ -498,6 +498,68 @@ bool pointInWorkspace(const MDHistoWorkspace *ws, const VMD &point) {
 /** Obtain coordinates for a line plot through a MDWorkspace.
  * Cross the workspace from start to end points, recording the signal along the
  *line.
+ *
+ * @param start :: coordinates of the start point of the line
+ * @param end :: coordinates of the end point of the line
+ * @param normalize :: how to normalize the signal
+ * @param x :: linearly spaced points along the line between start and end.
+ * @param y :: is set to the normalized signal for each bin. Length = length(x)
+ *- 1
+ * @param e :: error vector for each bin.
+ */
+void MDHistoWorkspace::getLinePlot(const Mantid::Kernel::VMD &start,
+                                   const Mantid::Kernel::VMD &end,
+                                   Mantid::API::MDNormalization normalize,
+                                   std::vector<coord_t> &x,
+                                   std::vector<signal_t> &y,
+                                   std::vector<signal_t> &e) const {
+  // TODO: Don't use a fixed number of points later
+  size_t numPoints = 500;
+
+  VMD step = (end - start) / double(numPoints - 1);
+  double stepLength = step.norm();
```

Comment:
I think you can uise `const auto`, Also above you can use `const`, (but not `auto`)